test/WebJobs.Script.Tests/Binding/HttpBindingTests.cs

```diff
@@ -56,12 +52,45 @@ public void ParseResponseObject_ReturnsExpectedResult()
             headers = null;
             statusCode = StatusCodes.Status200OK;
             enableContentNegotiationResponse = false;
-            HttpBinding.ParseResponseObject(responseObject, ref content, out headers, out statusCode, out enableContentNegotiationResponse);
+            HttpBinding.ParseResponseObject(responseObject, ref content, out headers, out statusCode, out cookies, out enableContentNegotiationResponse);
 
             Assert.Equal("Test Body", content);
             Assert.Same(headers, headers);
             Assert.Equal(StatusCodes.Status202Accepted, statusCode);
             Assert.True(enableContentNegotiationResponse);
+            // No cookies found or set
+            Assert.True(cookies == null || !cookies.Any());
+        }
+
+        [Fact]
+        public void ParseResponseObject_WithCookies_ReturnsExpectedResult()
+        {
+            var cookieProperties = new Tuple<string, string, CookieOptions>("hello", "world", new CookieOptions()
+            {
+                Domain = "/",
+                MaxAge = TimeSpan.FromSeconds(60),
+                HttpOnly = true
+            });
+
+            IList<Tuple<string, string, CookieOptions>> cookieContents = new List<Tuple<string, string, CookieOptions>>()
+            {
+                cookieProperties
+            };
+
+            dynamic responseObject = new ExpandoObject();
+            responseObject.Body = "Test Body";
+            responseObject.Cookies = cookieContents;
+            responseObject.StatusCode = "202";  // verify string works as well
+
+            object content = null;
+            var statusCode = StatusCodes.Status200OK;
+            HttpBinding.ParseResponseObject(responseObject, ref content, out IDictionary<string, object> headers, out statusCode, out List<Tuple<string, string, CookieOptions>> cookies, out bool enableContentNegotiationResponse);
+
+            Assert.Equal("Test Body", content);
+            Assert.Same(cookieContents, cookies);
+            Assert.Equal(StatusCodes.Status202Accepted, statusCode);
+            Assert.False(enableContentNegotiationResponse);
+            Assert.Same(headers, headers);
```

Comment:
typo? headers?
Nit: change assert statements to be more explicit. For example, expectedHeaders actualHeaders